Assets/Scripts/Editor/Tools/CompileTime.cs

```diff
@@ -76,7 +76,7 @@ void Update()
 
         void OnGUI()
         {
-#if UNITY_2018_4_OR_NEWER
+#if UNITY_2020_1_OR_NEWER
```

Comment:
The macro change was based on [enterPlayModeOptionsEnabled first appeared in Unity doc](https://docs.unity3d.com/2020.1/Documentation/ScriptReference/EditorSettings-enterPlayModeOptionsEnabled.html)